src/main/java/com/amazon/dlic/auth/http/saml/Saml2SettingsProvider.java

```diff
@@ -145,7 +145,7 @@ private void initMisc(HashMap<String, Object> configProperties) {
 
     private void initSpEndpoints(HashMap<String, Object> configProperties) {
         configProperties.put(SettingsBuilder.SP_ASSERTION_CONSUMER_SERVICE_URL_PROPERTY_KEY,
-                this.buildKibanaAssertionConsumerEndpoint(this.opensearchSettings.get("kibana_url")));
+                this.buildDashboardsConsumerEndpoint(this.opensearchSettings.get("kibana_url")));
```

Comment:
I would suggest renaming this to buildAssertionConsumerEndpoint. The assertion consumer service (ACS) endpoint is a location to which the SSO tokens are sent. Generic name would be good here.

Reply:
Changed.